Wikipedia/View Controllers/WebView/WebViewController.m

```diff
@@ -2015,7 +2011,7 @@ - (void)languageSelected:(MWKLanguageLink*)langData sender:(LanguagesViewControl
     [self dismissViewControllerAnimated:YES completion:nil];
 }
 
-#pragma Wikipedia Zero alert dialogs
+#pragma mark Wikipedia Zero alert dialogs
```

Comment:
If you add "-" to be `#pragma mark - Wikipedia Zero alert dialogs`, then you get a nice underline as well.